models/nft/nft_mints.sql

```diff
@@ -108,5 +115,11 @@ FROM (
 )
 
 SELECT * FROM project_mints
+ {% if is_incremental() %}
```

Comment:
maybe too nitpicky: would these filters be more performant if moved into the CTEs above?

Reply:
no, definitely better moved above.